app/src/androidTest/java/net/skyscanner/backpack/dialog/BpkDialogTest.kt

```diff
@@ -168,7 +169,41 @@ class BpkDialogTest : BpkSnapshotTest() {
     record(dialog, asyncScreenshot)
   }
 
+  @Test
+  fun screenshotTestFlareDialog() {
+    val bitmap = Picasso.get().load("file:///android_asset/dialog_sample.jpg").get()
+    val asyncScreenshot = prepareForAsyncTest()
+
+    val dialog = BpkDialog(activity, BpkDialog.Style.FLARE).apply {
+      title = "You are going to Tokyo!"
+      description = "Your flight is all booked."
+      icon = BpkDialog.Icon(
+        R.drawable.bpk_tick,
+        context.getColor(R.color.bpkMonteverde),
+      )
+
+      image!!.setImageBitmap(bitmap)
+
+      addActionButton(
+        BpkButton(context).apply {
+          text = "Continue"
+        }
+      )
+
+      addActionButton(
+        BpkButton(context, BpkButton.Type.Secondary).apply {
+          text = "Skip"
+        }
+      )
+    }
+
+    record(dialog, asyncScreenshot)
+  }
+
   private fun record(dialog: BpkDialog, asyncScreenshot: AsyncSnapshot) {
+    // not ideal, but the scrollbar disappears too early when running on CI causing test failures if visible
+    dialog.window?.decorView?.findScrollView()?.scrollBarDefaultDelayBeforeFade = 5000
```

Comment:
I have no idea why it disappears on CI as the screenshot is taken immediately when running locally. I don't like the workaround, but don't have a better idea

Reply:
Just to make sure I understand - the test is flaky because the scroll bar is sometimes visible and sometimes not. This guarantees that the scrollbar will always be visible?

Reply:
On CI it seems to always be invisible with the default timeout, whereas locally it is visible. I can't see what's happening, but my suspicion is that on CI the test takes longer to run and the scrollbar disappears before the screenshot is taken. Increasing the timeout in tests ensures that it is always visible

Reply:
got it 👍